package.json

```diff
@@ -30,8 +30,9 @@
     "color-convert": "^2.0.1",
     "contentlayer": "^0.2.5",
     "formik": "^2.2.9",
+    "framer-motion": "^7.6.1",
```

Comment:
Had to add this as a dependency, b/c it was only a peer dependency so it wouldn't work with package managers on webcontainer. Not sure if this is the right version though?